tools/esdlcmd/esdlcmd_common.hpp

```diff
@@ -325,10 +326,14 @@ static bool getCurrentFolder(StringBuffer & path)
 
 static bool getComponentFilesRelPathFromBin(StringBuffer & path)
 {
-    if (getCurrentFolder(path))
+    StringBuffer s;
```

Comment:
This new var prevents us from editing path until we can confirm path is valid, but could we just do a path.clear in the fail case?